astropy/utils/data.py

```diff
@@ -1370,3 +1370,27 @@ def _release_download_cache_lock():
         msg = 'Error releasing lock. "{0}" either does not exist or is not ' +\
               'a directory.'
         raise RuntimeError(msg.format(lockdir))
+
+
+def _urls_on_shelf():
```

Comment:
Perfect. I was thinking too narrowly about who would care, but I suppose so many affiliated packages use the download utils that this is probably useful for lots of people.